src/seedu/addressbook/data/person/ReadOnlyPerson.java

```diff
@@ -39,17 +39,17 @@ default String getAsTextShowAll() {
         final StringBuilder builder = new StringBuilder();
         final String detailIsPrivate = "(private) ";
         builder.append(getName())
-                .append(" Phone: ");
+                .append(" " + getPrintableString(this.getPhone()));
```

Comment:
New behavior will be different from the previous. I suspect each value will be printed twice. Did you test the output?